app/ts/components/ui/Grid.tsx

```diff
@@ -0,0 +1,24 @@
+import { ComponentChildren, JSX } from 'preact'
+import { mergeCSSClasses } from '../ui-utils.js'
+
+type GridProps = {
+	children: ComponentChildren
+	class?: string
+	style?: JSX.HTMLAttributes<HTMLDivElement>['style']
+}
+
+export const Grid = (props: GridProps) => {
+	const className = props.class ? mergeCSSClasses('grid', props.class) : 'grid'
+	return <div class = { className } style = { props.style }>{ props.children }</div>
+}
+
+type DataProps = {
+	children: ComponentChildren,
+	class?:string
```

Comment:
```suggestion
	class?: string
```

Reply:
the Grid component is no longer used, replaced with just `grid` CSS classname as there is no dynamic props to work with that makes any significant difference from a regular element